aidial_assistant/utils/exceptions.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from functools import wraps
+
+from aidial_sdk import HTTPException
+from openai import OpenAIError
+
+logger = logging.getLogger(__name__)
+
+
+class RequestParameterValidationError(Exception):
+    def __init__(self, message: str, param: str, *args: object) -> None:
+        super().__init__(message, *args)
+        self._param = param
+
+    @property
+    def param(self) -> str:
+        return self._param
+
+
+def _to_http_exception(e: Exception) -> HTTPException:
+    if isinstance(e, RequestParameterValidationError):
+        return HTTPException(
+            message=str(e),
+            status_code=422,
+            type="invalid_request_error",
+            param=e.param,
+        )
+
+    if isinstance(e, OpenAIError) and e.error:
+        http_status = e.http_status or 500
+        if e.error:
```

Comment:
It must be always True, given the check two lines above.

Reply:
Fixed and added tests.